node/src/accountant/mod.rs

```diff
@@ -3523,6 +4065,73 @@ mod tests {
         log_handler.exists_log_containing("DEBUG: Accountant: Found 2 pending payables to process");
     }
 
+    #[test]
+    fn start_scan_early_stop_for_pending_payables_if_initial_pending_payable_scan_and_no_wallet() {
+        let pending_payables_notify_later_params_arc = Arc::new(Mutex::new(vec![]));
+        let new_payables_notify_params_arc = Arc::new(Mutex::new(vec![]));
+        let mut subject = AccountantBuilder::default().build();
+        subject.scan_schedulers.pending_payable.handle = Box::new(
+            NotifyLaterHandleMock::default()
+                .notify_later_params(&pending_payables_notify_later_params_arc)
+                .stop_system_on_count_received(1),
+        );
+        subject.scan_schedulers.pending_payable.interval = Duration::from_secs(60);
+        subject.scan_schedulers.payable.new_payable_notify =
+            Box::new(NotifyHandleMock::default().notify_params(&new_payables_notify_params_arc));
+        let system = System::new("test");
+        let subject_addr = subject.start();
```

Comment:
I was wondering if it would be possible to explicitly represent the absence of a wallet.